02-introducing-survey-data.Rmd

```diff
@@ -1,2 +1,127 @@
+
+<!-- Put this in as an image instead of as a block quote, or need to add in an open-ended box for write in after the last option. -->
+> What animal do you prefer to have as a pet?
+>
+<ul class="ro">
+  <li class="ro">Dogs</li>
+  <li class="ro">Cats</li>
+  <li class="ro">Other, please specify:</li>
+</ul>
+
+Researchers can then code the responses from the open-ended box and get a better understanding of the respondent's choice of preferred pet.  Interpreting this question becomes easier as researchers no longer need to qualify the results with the choices provided.
+
+This is a very simple example of how the question presentation and options can impact the findings.  More complex topics and questions will need researchers to thoroughly consider how to mitigate any impacts from the presentation, formatting, wording, and other aspects.  As survey analysts, reviewing not only the data but also the wording of the questions is crucial to ensure the results are presented in a manner consistent with the question asked.
+
+### Data Collection
+
+Much of data collection relies on what is available on the sample frame (see \@ref(c05)).  For example, if our sampling frame includes email address, we could send email to our selected sample members to convince them to complete a web survey. While in this example the contact mode and the survey mode are similar (email and web), they do not have to be.  For example, we could mail respondents a letter with a link to access a web survey. However, it is important to make access to the survey as easy as possible for sample members to reduce burden and increase response rates.
+
+Understanding how sample members were contacted is important for determining the correct population that the data references.  One way researchers can contact respondents is using an address-based sampling (ABS) frame.  This frame is a list of addresses, so by nature, the survey is a household survey and will not incorporate respondents from those who do not have an address (e.g., homeless).  Additionally, using an address based approach will most often result in a population of _households_.  Surveys can combat this by asking the households to select an individual from the household to complete the survey.  This is called within household selection, however, there is no guarantee that the person completing the survey is the one that _should_ have completed the survey giving the within household selection criteria.
+
+This is just one example of how the data collection mode can impact the interpretation of survey results.  Other modes also have their own considerations, and survey analysts should read the survey documentation carefully (see \@ref(c03)) to have a better understanding of who was sampled and what the target population should be.
+
+## Error Sources and Weights
```

Comment:
OK, this is where weighting comes in but let's discuss restructuring so that sections flow based off of diagram but TSE as its own section not completely tied to weights.